oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandler.java

```diff
@@ -409,28 +429,15 @@ public APIGatewayProxyResponseEvent authoriseRequestHandler(
                 && browserSessionIdFromSession.isPresent()
                 && !Objects.equals(browserSessionIdFromSession, browserSessionIdFromCookie)) {
             sessionWithValidBrowserSessionId = Optional.empty();
+            auditService.submitAuditEvent(
+                    OidcAuditableEvent.AUTHORISATION_INITIATED,
+                    authRequest.getClientID().getValue(),
+                    user,
+                    pair("new_authentication_required", true));
         } else {
```

Comment:
Is this logically the same, without the `else`?

```
        Optional<Session> sessionWithValidBrowserSessionId = session;
        if (configurationService.isBrowserSessionCookieEnabled()
                && configurationService.isSignOutOnBrowserCloseEnabled()
                && browserSessionIdFromSession.isPresent()
                && !Objects.equals(browserSessionIdFromSession, browserSessionIdFromCookie)) {
            sessionWithValidBrowserSessionId = Optional.empty();
            auditService.submitAuditEvent(
                    OidcAuditableEvent.AUTHORISATION_INITIATED,
                    authRequest.getClientID().getValue(),
                    user,
                    pair("new_authentication_required", true));
        }
```

Reply:
Yea, good spot!